lib/optimizely.rb

```diff
@@ -388,39 +388,37 @@ def get_feature_variable_for_type(feature_flag_key, variable_key, variable_type,
 
```

Comment:
The document says:

> Make sure we return Null from `getFeatureVariableValueForType` when any of the following parameters are Null or empty
> ●	featureKey
> ●	variableKey
> ●	userId

Please add checks for variablekey aur userid too. See PHP for reference. And add Unit tests as well

Reply:
Please update the method def

> # Returns nil if the feature flag or variable are not found.
Add userId not found. And when variable type is mismatch

Reply:
Done.

Reply:
@rashidsp Please update the method def as well